api/v1alpha1/status.go

```diff
@@ -151,6 +151,9 @@ type OperationStatus struct {
 	LastSuccessfulTime *metav1.Time `json:"lastSuccessfulTime,omitempty"`
 	// WaitingStatus stores information about waiting operation.
 	WaitingFor WaitingStatus `json:"waitingFor,omitempty"`
+
+	// NodeLabels defines the node labels for operation pods
+	NodeLabels map[string]string `json:"nodeLabels,omitempty"`
```

Comment:
Can we just use `*v1.NodeSelectorTerm`, it can simplify the logic as we don't need `PreferredDuringSchedulingIgnoredDuringExecution` and an array of `v1.NodeSelectorTerm` in `*corev1.NodeAffinity` .

Reply:
@cheyang WDYT?

Reply:
I prefer `*corev1.NodeAffinity` because :

1. ** Consistency**：If the configuration and codebase already utilizes NodeAffinity,  maintaining consistency might reduce the learning curve and unify configuration patterns.
2. ** Extensibility**: If there's a possibility that more complex scheduling logic might be required in the future, starting with corev1.NodeAffinity could make such extensions more seamless.

@TrafalgarZZZ Your comments are welcome.

Reply:
ok, use *corev1.NodeAffinity.